packages/prism-react-renderer/src/components/__tests__/Highlight.test.tsx

```diff
@@ -14,15 +15,14 @@ describe("<Highlight />", () => {
   describe("snapshots", () => {
     it("renders correctly", () => {
       const { container } = render(
-        <Highlight {...defaultProps} code={exampleCode} language="jsx">
```

Comment:
Having to spread `defaultProps` as a way to use this library is not ergonomic, we're updating the library to automatically use the provided Prism and theme below.